packages/core/src/features/genomic/genesFrequencyChartSlice.ts

```diff
@@ -25,12 +25,12 @@ const GeneMutationFrequencyQuery = `
       geneFrequencyChartViewer: viewer {
         explore {
           cases {
-            hits(first: 0, case_filters: $caseFilters, filters: $geneFrequencyChart_filters) {
+            hits(first: 0, case_filters: $geneCaseFilter) {
               total
             }
           }
           genes {
-            hits(first: $geneFrequencyChart_size, offset: $geneFrequencyChart_offset, case_filters: $caseFilters, filters: $geneFrequencyChart_filters, score: $score) {
+            hits(first: $geneFrequencyChart_size, offset: $geneFrequencyChart_offset, case_filters: $geneCaseFilter, filters: $geneFrequencyChart_filters, score: $score) {
```

Comment:
I just wanted to double-check this change. It seems to be correct (not affect any of the past behavior). If you don't think so please let me know.
cc. @wteouchicago @craigrbarnes

Reply:
my code-fu isn't sufficient to know for sure, but from testing the graph, I didn't spot anything wrong.

Reply:
That is correct.